README.md

```diff
@@ -1,7 +1,7 @@
-# Run0
+# Kaps
 
-<img src="https://img.shields.io/github/workflow/status/virt-do/run0/run0%20build%20and%20unit%20tests?style=for-the-badge" />
+<img src="https://img.shields.io/github/workflow/status/virt-do/kaps/run0%20build%20and%20unit%20tests?style=for-the-badge" />
 
-`run0` is an experimental OCI container runtime written in Rust. The project aims to provide a performant, intuitive and OCI-compliant CLI allowing users to run & manage containers. 
+`kaps` is an experimental OCI container runtime written in Rust. The project aims to provide a performant, intuitive and OCI-compliant CLI allowing users to run & manage containers. 
 
-**Project is experimental and should not be used in any production systems.**
\ No newline at end of file
+**Project is experimental and should not be used in any production systems.**
```

Comment:
Can you please remove that change, it's unrelated.

Reply:
It's done.